docs/tutorials/presentations-exchange/presentations-exchange.postman_collection.json

```diff
@@ -351,7 +351,6 @@
 							"// pm.collectionVariables.get(\"credential_issuer_id\").",
 							"pm.test(\"`credential_issuer_id` persisted to collectionVariables\", function() {",
 							"    const { alsoKnownAs } = pm.response.json().didDocument;",
-							"    pm.collectionVariables.set(\"credential_issuer_id\", alsoKnownAs[1]);",
```

Comment:
this seems like a correct change, given what is written here: 
https://github.com/w3c-ccg/traceability-interop/blob/main/docs/discovery.md